tests/Helper/ScriptsTest.php

```diff
@@ -25,12 +25,14 @@ public function test()
 
         $scripts->add('/js/last.js', 150);
         $scripts->add('/js/first.js', 50);
+        $scripts->add('/js/attr.js', 100, array('foo' => true, 'bar' => 'baz'));
```

Comment:
cool. Can you add a test case for it and you can merge yourself. Or ping me. I will do the same.